Resources/deployments/function-app-resources.json

```diff
@@ -81,7 +81,7 @@
             },
             {
               "name": "CustomerSearchIndexName",
-              "value": "customer-search-index"
+              "value": "customer-search-index-v2"
```

Comment:
This index is intended for the Search API not the Customers API.  Will discuss with the dev